guardian/prosemirror-elements
# chore: Rename fieldName to fieldType

## What does this change?

Our codebase has two sorts of things called `fieldName`:

- The name of the field as defined by devs making elements
- The type of the field, e.g. text, richText

This PR renames the latter to `fieldType`, and renames the _existing_ `fieldType` (referring to the internal modelling prosemirror-elements does to represent values) to `fieldContentType`.

## How to test

Passing tests should indicate the refactor is OK.

Does the naming make sense? Anything to change?
